src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -63,6 +65,10 @@ export default class EthereumBlockchainService implements BlockchainService {
   public async sendTransaction(rootCid: CID): Promise<Transaction> {
     const wallet = new ethers.Wallet(config.blockchain.connectors.ethereum.account.privateKey, this.provider);
     const walletBalance = await this.provider.getBalance(wallet.address);
+    logMetric.ethereum({
```

Comment:
This is actually a metric in my mind, since it's information about global state at a specific point in time